packages/manager/modules/pci/src/projects/project/instances/instance/reinstall/reinstall.controller.js

```diff
@@ -44,4 +53,16 @@ export default class PciInstanceReinstallController {
         this.isLoading = false;
       });
   }
+
+  /**
+   * This function is used in case where customer image is deprecated
```

Comment:
```suggestion
   * This function is used in case customer image is deprecated
```

Reply:
@mohammed-zahaf No it's definitely not correct in that case 
```suggestion
   * This function is used in the case where customer image is deprecated
```
is